client/src/components/CreatePost.tsx

```diff
@@ -22,6 +22,16 @@ import { LocationPickerMap } from './LocationMap';
 
 const api = new ServerApi();
 
+interface FormState {
+  title: string;
+  body: string;
+  file: File | undefined;
+  tags: string[];
+  capacity: number;
+  location: string;
+  coords: { lat: number; lng: number };
+}
+
 function CreatePost() {
   const [openPopup, setOpenPopup] = useState(false); // for preview popup
   const [isAlertOpen, showAlert] = useState(false); // for snackbar
```

Comment:
Show an alert if they attempt to upload an invalid file type

Reply:
Do we have a standard alert component?

Reply:
There's something in `CreatePost` that handles error messages in the snackbar. I think `setMsg` and `showAlert` can be used here

Reply:
That would require a refactoring. We need to have a standard alert component shared across different pages. Leaving as a todo for now